nautobot/core/templates/inc/media.html

```diff
@@ -2,6 +2,56 @@
 {% load helpers %}
 {% load plugins %}
 
+    <style>
+        @font-face {
```

Comment:
It's hard to say how many variants are actually in use, but I know at least 4 of 6 are. It doesn't hurt to have them all served though, because browsers download a certain font only when it's actually in use on a given page, unused fonts are simply omitted, not causing any unnecessary network load.